graphene/core/schema.py

```diff
@@ -69,8 +67,7 @@ def T(self, _type):
     @property
     def executor(self):
         if not self._executor:
-            self._executor = Executor(
-                [SynchronousExecutionMiddleware()], map_type=OrderedDict)
+            self._executor = SyncExecutor()
```

Comment:
We were using `self._executor` because the default used in graphql-core in versions previous to `0.5.0` was not using a OrderedDict by default.
However this is no longer necessary, as now we can use the default chosen by `graphql-core`.
Just doing `return self._executor` should work fine :)